### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by XsidX**
For this project I:
-Have set up a new project with webpack.
-Created an index.html file and write your HTML markup here. 
-Created an index.js file and set an array of some simple to-do tasks (array of objects). Each task contains three keys:
description [string].
completed [bool].
index: [number].
-Have written a function to iterate over the tasks array and populate an HTML list item element for each task.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    >
+      <path
+        d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
+        fill="#b6b6b8"
+      ></path>
+    </svg>
+  `;
+
+  form.appendChild(listItem);
+});
```
Comment (julie-ify) [inline src/index.js]: - [ ] Kindly add this functionality in a function and call the function when the page loads. You can check this [link](https://www.w3schools.com/jsref/event_onload.asp#:~:text=The%20onload%20event%20occurs%20when,CSS%20files%2C%20etc.) to learn how to do this.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# To Do list with Webpack
 
+> n this project, I build a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
```
Comment (julie-ify) [inline README.md]: - [ ] There is a typo error at the beginning of this sentence, you have `n` but I guess you wanted to type `In`. Kindly fix it.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @XsidX,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️  
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @XsidX, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**